OpenApiQuery/OpenApiQuery.csproj

```diff
@@ -1,7 +1,7 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
 
     <PropertyGroup>
-        <TargetFramework>netcoreapp3.0</TargetFramework>
+        <TargetFramework>netstandard2.1</TargetFramework>
```

Comment:
Targeting netcoreapp for libraries is considered an antipattern

Reply:
At the times of ASP.net core 2.2 I would have fully agreed here and I would have targeted the .net standard. But with ASP.net core 3, .net core 3 and the path to .net 5 things seemed to change quite a bit. 

There are no ASP.net core 3.x nuget packages that you could target directly. You can see clearly the impact on your other changes below where you need to pull the 2.2 packages from nuget. I also still not fully get why Microsoft tends to re-invent the wheel with every version. The ASP.net core libraries should just have been published as nuget packages targeting netstandard2.1 but this seems not to be the case. For ASP.net core 3.x you need to target netcoreapp3.x and use the corresponding SDK or framework reference: 

https://docs.microsoft.com/en-us/aspnet/core/fundamentals/target-aspnetcore?view=aspnetcore-3.1&tabs=visual-studio

I agree that the ".netcore**app**3.x" moniker might be misleading, but it seems to be the right way of targeting ASP.net core 3:

https://github.com/OData/WebApi/blob/8de67a7e4e40973f3a03227863069a9ec14cbdab/src/Microsoft.AspNetCore.OData/Microsoft.AspNetCore.OData.csproj#L3
https://github.com/domaindrivendev/Swashbuckle.AspNetCore/blob/8ff7b32838e46ca0f748bd3f443840fbb07fd45d/src/Swashbuckle.AspNetCore/Swashbuckle.AspNetCore.csproj#L7
https://github.com/microsoft/aspnet-api-versioning/blob/6ea139e77b82266ea1a8cc130b55725f8beeaa42/src/Microsoft.AspNetCore.Mvc.Versioning/Microsoft.AspNetCore.Mvc.Versioning.csproj#L6
https://github.com/IdentityServer/IdentityServer4/blob/8d8cce5c1963bc5ed4deac7fed9ecfe04808897b/src/IdentityServer4/src/IdentityServer4.csproj#L5

Reply:
But it compiles and works as netstandard2.1 library, why target netcoreapp3.1? In this case it cannot be referenced from others netstandard projects which is very bad.